### DISCUSSION TITLE
Bringing Narratives to life(itself) and Web3

### DISCUSSION THREAD

**Discussion by AndreasS2501** [rufuspollock/awesome-crypto-critique#190] [Ideas]
Hello Everyone :)

I recently heard this interview with Gavin Wood:
https://www.youtube.com/watch?v=gmDRvm84LRc

I found it to be interesting on multiple levels:, like some more personal background information about gavin.

But in particular I found the following 3 sections relevant :
1. Origins of the term web3  (https://www.youtube.com/watch?v=gmDRvm84LRc&t=1215s) Web3[20:15](https://www.youtube.com/watch?v=gmDRvm84LRc&t=1215s) Web3
Q:  "one of the questions i wanted to ask so a lot of the content that i'm creating is to like also bring in people or help
them understand kind of **what web 3 is** so first of all you i read you coined the word web 3.
GW:  "(...)  _less trust more truth_ crypto kind of blockchain definition that it has today so 
while if we want to say you know i think it's fair to say yes i did coin web 3 as it is defined today that would be me"

I haven't really checked with anyone else but it seems to _me_ that gavin is really contradicting himself quite a lot in this interview.  In the beginning he points out how he learns from others tries to listen etc. But later on he seems to be disappointed by how little innovation is going on. 

I do think gavin's devinition of web3 was a piece upon people reflected upon, in 2014. But by definition their reflections created something _different_ then what he had in mind for it (less trust more truth) . There are a lot of cultural narratives at work here (web3 space) (libertarians, techno optimism , solution-ism etc..) 

How many years would it need to realize Gavind Wood that if a term becomes so widely used as AI or Blockchain, the term itself ceases to have meaning on its own.

As I pointed out in a talk in 2018: https://andreass.gitlab.io/slides/blockchain_talk_2018/Blockchain2.html#/3
At a certain level Terms (like AI/Blockchain/Web3) simply become projection spaces of peoples hopes, dreams and fears.
This is also described here:
https://tokeneconomy.co/visions-of-ether-590858bf848e and even best visually summarized in this image from the article:
https://cdn-images-1.medium.com/max/2600/1*ZpdeufZXonF4IXP3NI9ssg.png

Thankfully this gaping contradiction is realized more and more  also show here : https://github.com/life-itself/web3/discussions/175#discussioncomment-3100369

When Gavin Wood was using "less trust more truth" it seems to be he definitely was missing a lot of feedback from his community but also others. 

2. Is there enough innovation going on in kusama (polkadots test net) [28:29](https://www.youtube.com/watch?v=gmDRvm84LRc&t=1709s) 

Which brings me to the second interesting question: 
Q:  "how do you think kusama is shaping up philosophically like do you think there's enough chaos going on are people moving fast enough are they trying to break things enough uh yeah what do you think about that in short?"
GW: "i would say no i do not think it is"

So there seems to be an apparent lack between what Gavin imagines and what culture is expressed through kusama.

3. Betrand Russel - A History of Western Philosophy - [11:59](https://www.youtube.com/watch?v=gmDRvm84LRc&t=719s) Morals and Philosophy

"the book is that i you know i had a very uh um instinctive um sort of um 
agreement with uh the principles of like enlightenment uh of the western enlightenment of enlightened enlightenment philosophy is kind of like the principles of like you know liberal values um of uh accountability uh coupled with responsibility um 
of uh uh sorry **accountability and uh and responsibility kind of being the same**

".. and there's being coupled with authority so it's like if you have the power then you need to be accountable you need to be responsible"

Being in contact with the awakening from the meaning crisis discord and also doing practices regularly there has really helped me personally to come into a better relationship with technology.  I do think in some sense that Gavin wood wishes to do good in the world. But it seems to me his perception of technology is at least kind of incomplete. 

I even think I would go so far that Gavin would need more trust in people and a culture which helps people to develop and maintain better relationships. 

All the web3/Blockchain stuff hinges on the correct assumption that you can crypt information in one place and send it to another person and in between that information is "safe". That may be true, but it changes a lot of assumptions about what kind of information you want to transmit this way. This seems to me like a very strange kind of escapism , even slightly paranoia:  Humans are too faulty, trust no one , just the code which was written by humans...

The reality is far the opposite! Yes humans are paradoxical but we also inherit a very nice balance between change and rest. When Gavin points out early in the interview how Ethereums "mistake" was that it didn't account for upgrades, it touches this exactly. 

When people are in a "normal" setting communication is simple referring to norms and established patterns of communication. 

Lets assume the following: our (western mainstream industrial) civilization has put itself into a corner where the mainstream cultural patterns are destructive to people and to the environment. 

To get out of this corner it would be necessary to build new cultures which bring us in better relationship with each other and the environment. For this one can use different technologies and methods , also as described here: 
https://ecosystem.lifeitself.us/

:)

I do think that gavin sees blockchains and his polkadot as a project to help with this cultural transformation. But if there is already a gap from his perspective to what the people are doing there (testnet kusama), then **maybe there is more needed than "just" software  or blockchains as a mechanism to create a cultural change.**

Focusing on communication, relationshsips, practices even rituals. I would refer to these as:  acknowledging the complexity of human beings but also that everything is connected. People are connected to each other and to the environment. 

I do see projects in the web3 space which acknowledge this complexity and try to improve on inter human communication and relationships. (e.g. here: https://twitter.com/0xAutodidact/status/1549459112546779144 )

There is more to be said here on scope and scale. 

To finish this part I want to put again the spotlight on how important it is to communicate values and go beyond simple phrases (less trust more truth). To humbly consider technology not as _the_ solution to our problems, but as a part of our complexity.
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

**Comment by rufuspollock** [comment]
> At a certain level Terms (like AI/Blockchain/Web3) simply become projection spaces of peoples hopes, dreams and fears.

This a brilliant and spot-on point! web3 etc becomes a kind of ideological rorschach test where what people see is mainly their projections: libertarians see freedom and reduction of the state, finance folks see no SEC etc, socialists see participatory tech and democratized governance etc.

> This seems to me like a very strange kind of escapism , even slightly paranoia: Humans are too faulty, trust no one , just the code which was written by humans...

👍

@AndreasS2501 if you were happy to tidy this seems another definite keeper for a substantive wiki entry.

**Reply by AndreasS2501** [reply]
Hi @rufuspollock I'm still on vacation, things should settle down in september a bit I will then attempt to tidy things up :)
